src/ArborX_LinearBVH.hpp

```diff
@@ -162,7 +163,13 @@ BasicBoundingVolumeHierarchy<MemorySpace, BoundingVolume, Enable>::
       "ArborX::BVH::BVH::calculate_scene_bounding_box");
 
   // determine the bounding box of the scene
-  Box bbox{};
+#if KOKKOS_VERSION >= 30700
+  using scene_bounding_box = ExperimentalHyperGeometry::Box<
+      GeometryTraits::dimension<bounding_volume_type>::value>;
+#else
+  using scene_bounding_box = Box;
+#endif
+  scene_bounding_box bbox{};
```

Comment:
```suggestion
#if KOKKOS_VERSION >= 30700
   using Box = ExperimentalHyperGeometry::Box<
       GeometryTraits::dimension<bounding_volume_type>::value>;
 #endif
   Box bbox{};
```
(same as in `BVH::query()`)